Add learning center walkthroughs (#16893)
docker/docs: get-started/run-your-own-container.md

---
title: Create and run your own container
keywords: get started, quick start, intro, concepts
description: Learn how to run a container from scratch
---

> In this guide you'll learn the basic steps to run a container from scratch. For this guide, you'll use a sample Node.js application, but it's not necessary to know Node.js.

Reviewer: ```suggestion
Follow this guide to learn the basic steps of running a container from scratch. This guide uses a sample Node.js application, but it's not necessary to know Node.js.
```
